wiki/Main_Page/en.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: "main_page"
+blurb: "The osu! wiki is a project based on the contributions of people from osu!. The end goal is that it replaces the old FAQ section, and allows it to become a much more exhaustive resource. However, to make this possible, we need help from everyone willing to contribute!<br>You can use your osu! account to start contributing!"
```

Comment:
should be normal markdown at the beginning of article

Reply:
Do we allow for HTML in the markdown source? Because right now this text how it is on the page is centered, which would have to be applied in CSS on the web side, and due to how our markdown parser works, this would apply to all paragraphs.

Reply:
this page only has centered content, right? that should be fine. the news page already has custom styles for its markdown